### ISSUE TITLE
Global style for buttons

### ISSUE THREAD

**Issue by peterpinkster** [soflyy/oxygen-bugs-and-features#735]
**Describe the feature you'd like to see included in Oxygen.**
...Global style for buttons.  padding, color, shadow etc. 

**What are the use cases for this feature?**
Someone doesnt need to search the .ct-link-button in the source code to manipulates it. 

**Examples of this feature or functionality.**
*If another product has done it well, provide examples here.*
Brizy for example

**Comment by alFrame** [comment]
I might be missing something. But can't you add a class to your buttons and then style it globally under Settings -> Stylesheet using regular CSS?

**Comment by joecodino** [comment]
Why are there styling options for text links but not for buttons? Same as you can do for WooCommerce global settings...

**Comment by swinggraphics** [comment]
1. Button background colors (normal and hover) should probably inherit the color from Links / All.
2. Global Styles / Links / Buttons should have settings for text-align and padding, and for color and background-color in normal and hover states.

**Comment by xtechgamer735** [comment]
I'm astounded this isn't a feature. Looking to move to Oxygen for our agency and this is kinda a deal breaker. +1!

**Comment by jamescrabb** [comment]
> I might be missing something. But can't you add a class to your buttons and then style it globally under Settings -> Stylesheet using regular CSS?

I agree, use classes. That is all a global styles for buttons would be doing the UI

**Comment by FruitfulSystems** [comment]
Before anyone starts adding more global anything, I think maybe we should address the fact that when adding templates or composite items you have the heavy baggage of other global stylings being imported into your project. 

We need some sort of global merger system for this. Here is an example. 

when I import a footer from the financial design set it uses the financial design set color palette. Then I have to go in and manually adjust every element to use my global colors and not the financial globals. 

What should happen in my opinion is that once I import the footer, I should be given the option to map the global colors used in the component to my global colors that populate in my instance of said design component.

This mapping system also needs to be implemented for the global styles panel so I can change all colors used over to my global color system.

**Comment by swinggraphics** [comment]
@FruitfulSystems I think that sounds like a great improvement, and it should go in its own thread. It is a different scope from this request. Basic global styles for buttons is needed for everyone, regardless of using templates or composite events.
